frontend/src/views/ParticipantsList.js

```diff
@@ -0,0 +1,107 @@
+import React, { Fragment, useContext, useEffect } from "react"
+import { Link } from "react-router-dom"
+import { observer } from "mobx-react-lite"
+import { makeStyles } from "@material-ui/core/styles"
+import Table from "@material-ui/core/Table"
+// import TableRow from "@material-ui/core/TableRow"
```

Comment:
delete. Already hard enough to read all the imports as is without the comments.

Reply:
deleted with all the other commented code, just lest over from testing.